twindle-cli/twitter/utils/tweet-info.js

```diff
@@ -1,7 +1,28 @@
 const { writeFile } = require("fs").promises;
-let tweets = [];
+let tweets = { 
+    common : {
+      created_at: "",
+      count: "",
+      user: {
+        id: "",
+        username: "",
+        name: "",
+        profile_image_url: "",
+        description: ""
+      }
+    }, 
+    data : [] };
 
-const addTweet = (tweet) => tweets.push(tweet);
+const addTweet = (tweet) => tweets.data.push(tweet);
+
+const addCommon = (tweet, user) => {
```

Comment:
I think you the spread operator is better here, hence it is already shown in the original object which properties that it has.

```javascript
const addCommon = (tweet, user) => {
    tweets.common.user = { ...user };
    tweets.common.created_at = tweet.created_at;
}
```

something like this.

Reply:
I am not familiar with destructuring objects very much. But I will try it.. And I kept the names matching to precisely use it.. So if it works I will use it..